### ISSUE TITLE
[17.0] l10n_es: Taxes names are not user friendly

### ISSUE THREAD

**Issue by HaraldPanten** [odoo/odoo#167327]
### Odoo Version

- [ ] 15.0
- [ ] 16.0
- [X] 17.0
- [ ] Other (specify)

### Steps to Reproduce

Until version 16.0, taxes had more descriptive names. For Odoo users, it was very easy to identify which taxes should be applied in every sale/invoice.

Since version 17.0, taxes changed their names and they do not have user friendly details. For example:

![Captura de pantalla de 2024-05-30 12-20-03](https://github.com/odoo/odoo/assets/32449357/3b63a266-9e00-4e54-b4fd-766c6334c86f)

I know that the most common usage is the auto-detection by fiscal positions, but users should be able to identify if the applied taxes are correct or not. With the current names, it's hard to check it.

In the last Spanish OCA Days in Tenerife, after talking to some Odoo team members, they told us that taxes names, etc. in version 17.0 would be replaced by the old ones (or at least similar ones).

How's it going? Do you prefer us to open a new PR to help with that work?

@jco-odoo @pedrobaeza

**Comment by pedrobaeza** [comment]
Yes, the names have initials in English, confusing even more, so I vote for doing a change.

**Comment by jco-odoo** [comment]
@pedrobaeza @HaraldPanten No problem if we want to add a column name@es in the CSV to also have the short name in Spanish.  However, it is normal that users might need some time to adapt to the shorter versions.

**Comment by pedrobaeza** [comment]
OK to translate the short names to Spanish, but for me it's the same cryptic to find "21% G" (of "Goods") than "21% B" (of "Bienes"). Why this can't be translated to "21% Bienes"?

**Comment by jco-odoo** [comment]
@pedrobaeza Because this is a feature applied in all localizations to allow users to do "quick encoding".  Also, if you see in the screenshot, the Bienes are already visible.

**Comment by pedrobaeza** [comment]
And RC? And NS? It was just an example, but I think this is not very usable only with initials.

**Comment by HaraldPanten** [comment]
I agree with Pedro, on the other hand, there are "codes" not that short, for example:

- 0% EXEMPT Otros --> This one is longer than "21% Bienes", for example.
- EXEMPT Art.24 --> Longer as well
- 15% WHI worker cash --> This one has a "small description". Something similar as we could do with "21% Bienes"

Do you know what I mean?

Maybe we could use a quick encoding with a small description between brackets "( ... )", something like:

- 21% B (Bienes)
- 21% S (Servicios)
- 0% EU S (intracomunitario servicios)

We do not need really long names, but a couple or three words would be helpful.

WDYT? Could be this a good solution in order to satisfy both parts?

**Comment by HaraldPanten** [comment]
> I agree with Pedro, on the other hand there are "codes" not that short, for example:
> 
> * 0% EXEMPT Otros --> This one is longer than "21% Bienes", for example.
> * EXEMPT Art.24 --> Longer as well
> * 15% WHI worker cash --> This one has a "small description". Something similar as we could do with "21% Bienes"
> 
> Do you know what I mean?
> 
> Maybe we could use a quick encoding with a small description between brackets "( ... )", something like:
> 
> * 21% B (Bienes)
> * 21% S (Servicios)
> * 0% EU S (intracomunitario servicios)
> 
> We do not need really long names, but a couple or three words would be helpful.
> 
> WDYT? Could be this a good solution in order to satisfy both parts?

I think that doing this and considering that soon or late we will do "Decide which taxes should be active/inactive by default in the Spanish COA." --> https://github.com/odoo/odoo/pull/166499#issuecomment-2127628765

**Comment by jco-odoo** [comment]
It is also a matter of just keeping the names in the m2m tags short, but I can check further with @chklop next week.

**Comment by HaraldPanten** [comment]
> It is also a matter of just keeping the names in the m2m tags short, but I can check further with @chklop next week.

THX! Let's see if we could find an intermediate solution for this.

**Comment by HaraldPanten** [comment]
@jco-odoo Did you have any news about this?

**Comment by chklop** [comment]
Hey guys,

In 17.3, we've improved the look of the Tax search to do basically what you want to do. Tax Scopes are now displayed next to Tax names in the Tax Search, so there's no need to write them in the Tax names.
![image](https://github.com/odoo/odoo/assets/93644924/427a94fc-e301-4669-8779-cb3bce8b1769)

Also now, if you use "Search more", the Tax Description is displayed in the search, where we can provide a long, full description of the tax so the user understands the meaning and purpose of the tax.
![image](https://github.com/odoo/odoo/assets/93644924/b27862a0-8635-4251-b655-43b618becab4)

As @jco-odoo said, we need to keep the Tax Names as short as possible, otherwise they quickly get unusable. They should be **short codes** ; once the user understands them, they are much easier to work on a day-to-day basis.

For Spain, I think we should:
- Modify the codes to be Spanish. E.g. change G (for Goods) to B (for Bienes)
- We should put a lot of the more specific taxes in "Inactive" by default. There are too many taxes that too many people don't care about. Have a look at l10n_be for example, we have 71 taxes in the loca, and only 18 active by default. We should have the same for l10n_es, to make onboarding cleaner.
- Expand on our "Tag descriptions" to properly describe them and their usage so users can "discover" them in the "Help me" ?

**Comment by rafaelbn** [comment]
@moduon MT-6591 Analyzed during the last year, asking accountants and administrative staff, since the name is given to them, the users. We do not yet have a consensus that we hope to advance in July.
